### PR TITLE
add triage workflows

### PR DESCRIPTION
**Pull request by ryelo**
This will:
* Apply labels based on what files were edited
* Add a "release" label on every PR inside a package
* Add a comment on stale issues (30 days)
* add an "awaitingReview" label on PRs and will remove that and add "Approved" on 1 review (which can be changed)

Note: actions do not work on forks

### CODE REVIEW

**Comment by ryelo** [comment]
^ see it works 😄

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+notifications:
+  - packages/notifications/**/*
+remediations:
+  - packages/remediations/**/*
+sources:
+  - packages/sources/**/*
+translations:
+  - packages/translations/**/*
+utils:
+  - packages/utils/**/*
```
Comment (Hyperkid123) [inline .github/labeler.yml]: :no_good_man:

Comment (ryelo) [reply .github/labeler.yml]: Can you expand on this review? Not really sure what this means 😄

Comment (Hyperkid123) [reply .github/labeler.yml]: :D ok there missing new lines

**Review by karelhala — APPROVED** [review]

**Pull request merged**